model/riscv_fetch_rvfi.sail

```diff
@@ -13,12 +15,12 @@ function fetch() -> FetchResult = {
         TR_Failure(e, _) => F_Error(e, PC),
         TR_Address(_, _) => {
           let i = rvfi_instruction.rvfi_insn();
-          rvfi_exec->rvfi_insn()     = EXTZ(i);
+          rvfi_inst_data->rvfi_insn()   = EXTZ(i);
           /* TODO: should we write these even if they're not really registers? */
-          rvfi_exec->rvfi_rs1_data() = EXTZ(X(i[19 .. 15]));
-          rvfi_exec->rvfi_rs2_data() = EXTZ(X(i[24 .. 20]));
-          rvfi_exec->rvfi_rs1_addr() = sail_zero_extend(i[19 .. 15],8);
-          rvfi_exec->rvfi_rs2_addr() = sail_zero_extend(i[24 .. 20],8);
+          // rvfi_exec->rvfi_rs1_data() = EXTZ(X(i[19 .. 15]));
```

Comment:
Why are these lines commented out? Perhaps a comment to explain?

Reply:
They should be deleted, the values are set when actually writing the registers now not during decode.